views/guide.lavender

```diff
@@ -297,10 +297,13 @@ block content
 
                 if FALSE
                   div this isn't rendered
+
                 elseif FALSE
```

Comment:
@zlincoln I don't specifically remember trying that but feel like I tried every variation. 

I will experiment.